<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Creation</string>
  </property>
  <widget class="QPushButton" name="doneBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>560</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>C:/Users/lucp8023/.designer/backup</normaloff>C:/Users/lucp8023/.designer/backup</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>521</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPlainTextEdit" name="browseText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="browseBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>79</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="labelText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addLabelBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add to set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File Label</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File Location</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>20</y>
     <width>521</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dataset Name</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="nameText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dataTypeCombo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Sunbrick</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DW2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LBIC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PDS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PTI</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="showSetPlainText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>521</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameText</tabstop>
  <tabstop>labelText</tabstop>
  <tabstop>browseText</tabstop>
  <tabstop>showSetPlainText</tabstop>
  <tabstop>browseBtn</tabstop>
  <tabstop>addLabelBtn</tabstop>
  <tabstop>doneBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
